src/dumpContent.ts

```diff
@@ -88,6 +95,15 @@ function dumpManifest(objIds: string[]) {
   )
 }
 
+function preprocessObjData(objData: ObjData): ObjData {
+  // Ignore pisa_url - this should be set up in the new instance
+  if (objData._site_id === 'default' && objData._path === '/') {
+    delete objData.pisa_url
+  }
```

Comment:
* [ ] FYI The code could be simplified to drop everything named `pisa_url`. In my mental model we basically remove all blacklisted attributes everywhere. (The length of the list of names is one at the moment.)